### PR TITLE
Write the class byte convert

### PR DESCRIPTION
**Pull request by TheYisusGithub**

### CODE REVIEW

File: src/main/java/com/fundation/logic/useful/ByteConvert.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.useful;
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/useful/ByteConvert.java]: I think this class should be placed in "common" package

File: src/main/java/com/fundation/logic/useful/ByteConvert.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ *
+ * @author Jesus Menacho
+ * @version 1.0
+ */
+public class ByteConvert {
+
+    /**
+     *This method convert byte to Kb, Mb and Gb.
+     */
+    public String bytesConvert(String bytesNumber, String unitConversion) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/useful/ByteConvert.java]: This methods should return a Float data type.

File: src/main/java/com/fundation/logic/useful/ByteConvert.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            case "GB":
+                return Double.toString(((Double.parseDouble(bytesNumber) / unitBase) / unitBase) / unitBase);
+        }
+        return null;
+    }
+
+    /**
+     *This method from Kb, Mb and Gb to bytes.
+     */
+    public String anyConvertBytes(String unitConversion, String numberToConvert) {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/useful/ByteConvert.java]: This method should return a Float data type.

File: src/main/java/com/fundation/logic/useful/ByteConvert.java (hunk 1 above)
Comment (TheYisusGithub) [reply src/main/java/com/fundation/logic/useful/ByteConvert.java]: Ok that was correct.

File: src/main/java/com/fundation/logic/useful/ByteConvert.java (hunk 2 above)
Comment (TheYisusGithub) [reply src/main/java/com/fundation/logic/useful/ByteConvert.java]: Ok it was fixed.

File: src/main/java/com/fundation/logic/useful/ByteConvert.java (hunk 3 above)
Comment (TheYisusGithub) [reply src/main/java/com/fundation/logic/useful/ByteConvert.java]: It was fixed now return Float.

**Pull request merged**